src/app/lib/business/environment/environment.dart

```diff
@@ -2,12 +2,14 @@
 enum Environment {
   development,
   staging,
+  generatedStaging,
   production;
 
   @override
   String toString() => switch (this) {
         development => 'Development',
         staging => 'Staging',
+        generatedStaging => 'Generated_Staging',
```

Comment:
We need to make sure that this isn't part of the generated template code. It can be acheive by using `#if false`

Reply:
Not implemetend, only in build folder for now, see `RemoveCommentedOutBlocks` in powershell, will refactor it in the CLI tool.